modules/tinymce/CHANGELOG.md

```diff
@@ -57,6 +57,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed incorrect word breaks in menu dropdowns with scrollbar #TINY-8572
 - The `InsertLineBreak` command did not replace selected content #TINY-8458
 - Delete operations could behave incorrectly if the selection contains a `contenteditable="false"` element located at the edge of content #TINY-8729
+- The selection is not moved over a comment on `editor.focus()` if the first element is a comment #TINY-7817
```

Comment:
This isn't really the fix in the end... the fix is in the `editor.selection.normalize()` function and it's that it was placing the selection inside a comment node.

- [ ] Reword changelog to match the fix

Reply:
done